### DISCUSSION TITLE
Change the ios-src at runtime through JS

### DISCUSSION THREAD

**Discussion by Anburaj** [google/model-viewer#4335] [Q&A]
Try to change the model when the user clicks a button. I am doing this functionality through JS. Initially, `modelviewer` is loaded with `a.glb` model. Then I am changing the model using the following line of code.

`modelViewer.src = "b.glb"`

It is working fine in Android. In iOS, I don't know what is the line of code to change the USDZ file.
I tried with 
`modelViewer.ios-src = "b.usdz"`
and
`modelViewer.iosSrc = "b.usdz"`, But no luck.
Can you help me to change the ios src dynamically.

**Comment by danilokrulj** [comment]
You can do it like this 
modelViewer.setAttribute('ios-src', 'yourModel.usdz')
